src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -300,6 +300,13 @@ end
 
 set_array("inCollection[]")
 
+# hbz NZ
+
+if any_equal("003", "DE-605")
```

Comment:
> We can change the condition for this if necessary later.

Do you mean this condition? It checks for `DE-605`, but assigns `DE-655`.

Reply:
Yes it checks for DE-605 but DE-655 is the ISIL for ALMA-NZ: http://lobid.org/organisations/DE-655

And I used this as URI for the ALMA-NZ.

Reply:
As long as we're clear that this is definitely not the final condition. `003=DE-605` includes local (IZ) records.

Reply:
I agree I just used one of your suggestions from https://github.com/hbz/lobid-resources/issues/1677)

Reply:
I'm sorry, but this wasn't one of the suggestions. `003=DE-655` was.

Reply:
Thanks for the hint.